Getting-Started-with-Android-Development.md

```diff
@@ -0,0 +1,20 @@
+# Getting started with Android Development
+
+So, you want to get started with Android development? Let me take you through some basic steps you should go through.
+
+#### Tools
+At first, make sure you have all the tools in your system. The things you'll need to start building application for Android are:
```

Comment:
A word of caution - at FreecodeCamp, most campers are not used to the idea of having to set up their environment by installing things. We do all things in browser.

And installing Android specif stuff would take them about 2-3 hours; while having to download 1-2 GB of files. It's a high barrier to entry.

So, you might want to start-off with that cautionary note. Also link to some installation guide for Java, Android Studio etc.